be/src/docker-compose.override.yml

```diff
@@ -195,6 +195,9 @@ services:
       - EventBusRetryCount=5
       - SupportedCultures=${SUPPORTED_CULTURES}
       - DefaultCulture=${DEFAULT_CULTURE}
+      - SenderEmail=orders@eltap.com
```

Comment:
**suggestion:** let's move all of them from docker-compose to .env and put a variable here